ga4gh/application.wsgi

```diff
@@ -0,0 +1,3 @@
+from ga4gh.frontend import app as application
```

Comment:
Sorry, I just spotted this now as I was about to merge it. I don't think this file should be in the ga4gh directory, as it's not part of the Python package. Could you move this to deploy (or somewhere else) please?